src/tests/topp/MRMTransitionGroupPicker_1_output.featureXML

```diff
@@ -138,6 +145,7 @@
 					<UserParam type="string" name="native_id" value="60348_NSFVSYENFYQNHENAEIR/3_y15_2"/>
 					<UserParam type="float" name="peak_apex_int" value="2110"/>
 					<UserParam type="float" name="total_xic" value="15604"/>
+					<UserParam type="float" name="width_at_50" value="13.6500000000001"/>
```

Comment:
I also measured exactly 13.6, see 

![screenshot from 2018-09-28 12-17-25](https://user-images.githubusercontent.com/464014/46220649-c960dd00-c318-11e8-8d7f-35577aa9fc87.png)

Reply:
I looked at a few examples, and it was always on spot (given that it does not interpolate, but reports multiples of the swath-cycle time)